docs/java-rest/high-level/miscellaneous/x-pack-watcher-put-watch.asciidoc

```diff
@@ -0,0 +1,55 @@
+[[java-rest-high-x-pack-watcher-put-watch]]
```

Comment:
Im not sure i like this being in the misc folder. Ill get clarification on where it should go.

Reply:
cc @debadair @Sue-Gallagher

Reply:
after a chat w @debadair , lets go ahead and create a `watcher/` folder and put all the watcher docs into that.

Reply:
I created an `x-pack` folder and inside of that a `watcher` folder as I assume both of them will grow soonish?

Reply:
yup, or you will end up with conflicts as someone else will also be doing this :)